test/netcard_test.rb

```diff
@@ -69,9 +69,8 @@
 
 require "yast"
 
-include Yast
-include UIShortcuts
-include I18n
+include Yast::UIShortcuts
```

Comment:
UI shortcuts is by default now part of rspec helper so not needed - https://github.com/yast/yast-ruby-bindings/blob/master/src/ruby/yast/rspec/shortcuts.rb#L8